src/core/components/accordion/index.stories.tsx

```diff
@@ -1,30 +1,39 @@
 import { Accordion, AccordionProps, AccordionRow } from './index';
 
 export default {
-	title: 'Accordion',
+	title: 'Source/src-accordion/Accordion',
 	component: Accordion,
 	subcomponents: { AccordionRow },
 };
 
-export const Default = (args: AccordionProps) => (
+export const Demo = (args: AccordionProps) => (
 	<Accordion {...args}>
 		<AccordionRow label="Collecting from multiple newsagents">
 			Present your card to a newsagent each time you collect the paper.
 			The newsagent will scan your card and will be reimbursed for each
 			transaction automatically.
 		</AccordionRow>
-		<AccordionRow
-			label="Delivery from your retailer"
-			hideToggleLabel={false}
-		>
+		<AccordionRow label="Delivery from your retailer">
 			Simply give your preferred store / retailer the barcode printed on
 			your subscription letter.
 		</AccordionRow>
 	</Accordion>
 );
 
-Default.args = {
+Demo.args = {
 	hideToggleLabel: false,
 };
 
-Default.story = { name: 'default' };
+export const WithCTALabels = (args: AccordionProps) => (
+	<Accordion {...args} hideToggleLabel={false}>
+		<AccordionRow label="" />
```

Comment:
Do you think stories for Chromatic should have some dummy data too? I think it would be useful to confirm that the data is rendered correctly.

Reply:
ok :+1:

Reply:
fixed in 82d9147